src/pages/DirectMessage/hooks/useTextArea.ts

```diff
@@ -0,0 +1,93 @@
+import { useRef, useState } from "react"
+import useSendMessage from "./useSendMessage"
+import { useParams } from "react-router-dom"
+import { MessageInputProps } from "../components/MessageList/MessageInput/MessageInput"
+
+type EventType =
+  | React.KeyboardEvent<HTMLTextAreaElement>
+  | React.FormEvent<HTMLFormElement>
+
+const useTextArea = ({
+  scrollRef,
+  setMessageListHeight,
+}: MessageInputProps) => {
+  const [textValue, setTextValue] = useState("")
+  const { AlertModalComponent, sendMessage } = useSendMessage()
+  const { userId: othersUserId } = useParams()
+  const textRef = useRef<HTMLTextAreaElement>(null)
+
+  const resize = () => {
+    if (!textRef.current) {
+      return
+    }
+
+    textRef.current.style.height = "0px"
```

Comment:
```suggestion
const { current } = textRef
    if (!current) {
      return
    }

    current.style.height = "0px"
```

아래부분 전부 줄일 수 잇을것 같아요